bodhi/tests/server/scripts/test_check_policies.py

```diff
@@ -220,3 +220,33 @@ def test_unrestricted_policy(self):
                  'type': 'bodhi_update'}]}
         mock_greenwave.assert_called_once_with(config['greenwave_api_url'] + '/decision',
                                                expected_query)
+
+    @patch.dict(config, [('greenwave_api_url', 'http://domain.local')])
+    def test_archived_release_updates(self):
+        """Assert that updates for archived releases isn't being considered
+        by the script.
+        """
+        # Archive the F17 release
+        rel = self.db.query(models.Release).filter_by(name='F17').one()
+        rel.state = models.ReleaseState.archived
+        self.db.commit()
+
+        runner = testing.CliRunner()
+        update = self.db.query(models.Update).all()[0]
+        update.status = models.UpdateStatus.testing
+        self.db.commit()
+        with patch('bodhi.server.models.util.greenwave_api_post') as mock_greenwave:
+            greenwave_response = {
+                'policies_satisfied': True,
+                'summary': 'All tests passed',
+                'applicable_policies': ['taskotron_release_critical_tasks'],
+                'unsatisfied_requirements': []
+            }
+            mock_greenwave.return_value = greenwave_response
+            result = runner.invoke(check_policies.check, [])
+            self.assertEqual(result.exit_code, 0)
+            update = self.db.query(models.Update).filter(models.Update.id == update.id).one()
+            self.assertEqual(update.test_gating_status, models.TestGatingStatus.passed)
+            self.assertEqual(update.greenwave_summary_string, None)
```

Comment:
These three lines don't really check anything since this is both the starting state of the update, but is also the state the update would get to if greenwave were erroneously accessed. I suggest just making the greenwave access raise an Exception and then drop these three lines to keep the test shorter and more laser focused.